### ISSUE TITLE
Calendar - Mobile view - No way to scroll down to save event

### ISSUE THREAD

**Issue by ToeiRei** [nextcloud/calendar#1079]
### Steps to reproduce
1. Use a mobile browser - chrome, current version
2. Log into nextcloud
3. Add event to calendar

### Expected behaviour
I would expect to operate the calendar from a mobile device / tablet

### Actual behaviour

Cannot scroll down below cancel button to create the event - see screenshot
![image](https://user-images.githubusercontent.com/5990591/51531728-87ffba80-1e3e-11e9-9cfa-c62fc614a5b9.png)

### Server configuration

**Operating system**: Gentoo Linux

**Web server:** nginx/1.15.8

**Database:** MariaDB 10.3.11

**PHP version:** 7.2.14

**Server version:** 15.0.2

**Calendar version:** 1.6.4

**Updated from an older installed version or fresh install:** Updated

**Signing status (ownCloud/Nextcloud 9.0 and above):** 

```
No errors have been found.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your instance's installation folder
```

**Nextcloud configuration:**

```
not relevant
```

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

### Client configuration
**Browser:** Google Chrome

**Operating system:** Android 8.1

**CalDAV-clients:**

**Comment by klemens-u** [comment]
I have this very servere issue too!  It renders this app basically unusable.

**Comment by klemens-u** [comment]
I've looked into this a bit more, and it seems the problem occurs mainly on older systems/browsers.
The height calculation of the sidebar, and therefore the decision when to display scollbars, is the problem.

As a quickfix for mobile devices, I "unfixed" the sidebar with custom css.
This makes the app usable for my clients.

* Menu -> Apps -> Add/Enable "Custom CSS" app
* Menu -> Settings -> Administration -> Theming
* Add custom CSS and save:

```
    /* Fix mobile right sidebar issues (create, update buttons hidden) */
     @media only screen and (max-height: 760px) {

      #app-sidebar {
        height: auto; /* there are problems with the height calculation */
        width: auto; /* use max width on mobile */
        top: 0; /* surpress top margin when not fixed (see below */
      }

      .modal-content #app-sidebar {
        position: relative;      /* do not fix sidebar to right side */
      }
    }

```

**Comment by svenb1234** [comment]
I would not consider my Nexus 5x with Android 15.1 and the latest Chrome app version an old system. Yet the same bug applies! The bottom android toolbar hides the calendar buttons.

**Comment by rollanders** [comment]
I have this error on iPad 9.7 (2015) - both in Safari and Chrome, and on Huawei P20. I cannot create new events, but I can edit existing events.

**Comment by fwolfst** [comment]
> I've looked into this a bit more, and it seems the problem occurs mainly on older systems/browsers.
> The height calculation of the sidebar, and therefore the decision when to display scollbars, is the problem.
> 
> As a quickfix for mobile devices, I "unfixed" the sidebar with custom css.
> This makes the app usable for my clients.
> 
>     * Menu -> Apps -> Add/Enable "Custom CSS" app
> 
>     * Menu -> Settings -> Administration -> Theming
> 
>     * Add custom CSS and save:
> 
> 
> ```
> /* Fix mobile right sidebar issues (create, update buttons hidden) */
> @media only screen and (max-height: 760px) {
> ...
> }
> ```

Thanks @klemens-u . I think I have two improvements to propose:
- mention the bug in a comment, so that it is easier to remove this workaround on upstream fixes
- use `max-**device**-height` instead of `max-height`. I am no CSS-pro, but from my understanding this should "fix" the display on smaller screen desktop-browsers. Unfortunately I couldn't test the mobile view case. The point is: it was working fine on a small screen laptop before, but somewhat ugly after applying the custom css.

```
/* Fix mobile right sidebar issues (create, update buttons hidden) */
/* Works around this issue https://github.com/nextcloud/calendar/issues/1079 */
/* Once this issue is solved, you can remove this modification */
@media only screen and (max-device-height: 760px) {

      #app-sidebar {
        height: auto; /* there are problems with the height calculation */
        width: auto; /* use max width on mobile */
        top: 0; /* surpress top margin when not fixed (see below */
      }

      .modal-content #app-sidebar {
        position: relative;      /* do not fix sidebar to right side */
      }
}
```

**Comment by georgehrke** [comment]
This was fixed in #926 

![5FB71565-2C34-455E-B9A6-EBDEBDB3D9CB](https://user-images.githubusercontent.com/1250540/67160254-39d90280-f34f-11e9-98e1-c5abd8054ec4.png)
